### PR TITLE
Update Documentation following TF-lib documentation

### PR DESCRIPTION
**Pull request by jatkinson1000**
Typos and infelicities.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +55,7 @@ To build and install the library:
     git clone https://github.com/Cambridge-ICCS/fortran-pytorch-lib.git
     ```
     to clone via https.  
-2. Navigate into the library source directory by running:  
+2. Navigate into the library directory by running:  
     ```
     cd fortran-pytorch-lib/fortran-pytorch-lib/
```
Comment (TomMelt) [inline README.md:60]: This line needs to change now because we have reorganised and renamed the FTorch repo.

**Review by TomMelt — CHANGES_REQUESTED** [review]
I am happy for this to be merged if we update the library installation instructions

File: README.md (hunk 1 above)
Comment (jatkinson1000) [reply README.md]: This got covered in the renaming updates, so I've just merged main into this branch which resolves it.

Comment (TomMelt) [reply README.md]: perfect. thank you :)

Comment (jatkinson1000) [reply README.md]: If you can clear it I'll go ahead and merge.

**Review by TomMelt — APPROVED** [review]
LGTM

**Pull request merged**